airbyte-integrations/connectors/source-mssql/build.gradle

```diff
@@ -6,7 +6,7 @@ plugins {
 airbyteJavaConnector {
     cdkVersionRequired = '0.7.2'
     features = ['db-sources']
-    useLocalCdk = true
+    useLocalCdk = false
```

Comment:
revert?

Reply:
@stephane-airbyte if I revert this then the build would start breaking, I have deleted code from cdk that mssql is using, so if use the local cdk, the mssql build will fail with compilation error. I dont know why it was set to true at the first place? For mysql, postgres, mongo it's set to false

Reply:
oh, yeah, that's on me. Let me push that change separately. Sorry about that!